src/web/layouts/CommentLayout.tsx

```diff
@@ -209,6 +209,10 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
 
     const contributorTag = CAPI.tags.find(tag => tag.type === 'Contributor');
     const avatarUrl = contributorTag && contributorTag.bylineImageUrl;
+    const onlyOneContributor: boolean =
+        CAPI.tags.filter(tag => tag.type === 'Contributor').length === 1;
```

Comment:
```suggestion
        CAPI.tags.filter(tag => tag.type === 'Contributor').length >= 1;
```
in case there is more than one of the same tag (just a precaution)

Reply:
Actually, we only want to return true when the count is exactly 1